src/metabase/server/middleware/session.clj

```diff
@@ -356,10 +356,23 @@
   ;; The amount is nillable.
   (deferred-tru "Time before inactive users are logged out. By default, sessions last indefinitely.")
   :type       :json
-  :default    nil)
+  :default    nil
+  :setter     (fn [new-value]
+                (when (some? new-value)
```

Comment:
Should we also be checking that the value is positive? It seems like a negative number would cause problems as well.

Reply:
Yeah, updated https://github.com/metabase/metabase/pull/27347/commits/53a4854231d1f5331bf18b20350cc24c05860063